AsyncDisplayKit/ASNetworkImageNode.mm

```diff
@@ -211,9 +212,9 @@ - (void)displayWillStart
 - (void)visibilityDidChange:(BOOL)isVisible
 {
   [super visibilityDidChange:isVisible];
-  
+
+  ASDN::MutexLocker l(_lock);
```

Comment:
Same question here, this lock will be held throughout the call to _updateProgress… yes?